### PR TITLE
Add .travis.yml.

### PR DESCRIPTION
**Pull request by lexrus**
Travis, a popular CI service, can help to test every single PR / commit automatically. All we need to do is add the `.travis.yml` to the root and enable the service for this repo in your Travis dashboard.
https://travis-ci.org

### CODE REVIEW

**Comment by JakeLin** [comment]
@lexrus You are awesome, I haven't used Travis before, I use Jenkins and am considering to use Buildkite to replace Jenkins.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+osx_image: xcode7.2
+language: objective-c
```
Comment (JakeLin) [inline .travis.yml:2]: does it matter to use objective-c instead of swift?

**Comment by JakeLin** [comment]
I have signed in to Travis with my github account, will set it up  <del>tomorrow</del> today (12:00 already 😱) on https://travis-ci.org/JakeLin/IBAnimatable

File: .travis.yml (hunk 1 above)
Comment (lexrus) [inline .travis.yml:2]: `language: objective-c`
This line is optional somehow.
https://blog.travis-ci.com/2015-12-10-xcode-72-is-ready

**Comment by lexrus** [comment]
Cool.
Good night.

**Comment by JakeLin** [comment]
Travis is working now, thanks.

**Pull request merged**